satpy/readers/fci_l1c_fdhsi.py

```diff
@@ -257,16 +268,16 @@ def calibrate(self, data, key, measured, root):
 
     def _ir_calibrate(self, radiance, measured, root):
         """IR channel calibration."""
-        coef = self[measured + "/radiance_unit_conversion_coefficient"]
-        wl_c = self[root + "/central_wavelength_actual"]
+        # using the method from EUM/MET/TEN/11/0569 and PUG
```

Comment:
Do we have a link to these documents in the references at the top of the file?

Reply:
Only for the PUG, good point. Added the link and expanded the documentation slightly.